LanternExtractor/EQ/ClientDataCopier.cs

```diff
@@ -53,12 +53,12 @@ private static List<string> GetStaticFilePaths(Settings settings)
         private static string GetDestinationPath(string rootFolder, string sourceFilePath)
         {
             var sourceFileName = Path.GetFileName(sourceFilePath);
-            return Path.Combine(rootFolder + StaticDirectory, sourceFileName);
+            return Path.Combine(rootFolder + ClientDataDirectory, sourceFileName);
         }
 
-        private static bool IsInvalidName(string archiveName)
+        private static bool IsInvalidName(string fileName)
         {
-            return !EqFileHelper.IsStaticArchive(archiveName) && archiveName != "all";
+            return !EqFileHelper.IsClientDataFile(fileName) && fileName != "clientdata";
```

Comment:
I think this should be changed back to `!= "all"`. IsClientDataFile already handles `"clientdata"`